msteams-platform/sbs-teams-app-test-tool.yml

```diff
@@ -0,0 +1,351 @@
+  nextTutorialTitle: Teams App Test Tool
+  ms.custom: mvc
+  ms.localizationpriority: high
+items:
+- durationInMinutes: 1
+  content: |
+    Start Microsoft Teams app development with your Teams AI chat bot app and debug with the Teams App Test Tool (Test Tool). Test Tool makes debugging bot-based apps effortless. You can chat with your bot and see its messages and Adaptive Cards as they appear in Teams. You don't need a Microsoft 365 developer account, tunneling, or Teams app and bot registration to use Test Tool. You can also mock an activity in Test Tool using activity triggers.
+
+    > [!NOTE]
+    >
+    > * Teams App Test Tool is available only in the latest prerelease version of the Teams Toolkit.
+    > * Teams App Test Tool is supported only for desktop and web clients.
+
+    This step-by-step guide helps you to build AI chat bot using Teams Toolkit and debug AI chat bot with the Test Tool. You'll see the following output after you've completed this guide, where the user can access and use the AI chat bot:
+
+    :::image type="content" source="~/assets/images/teams-toolkit-v2/debug/test-tool.png" lightbox="~/assets/images/teams-toolkit-v2/debug/test-tool.png" alt-text="Screenshot shows the bot open in Test Tool.":::
+    
+- title: Prerequisites
+  durationInMinutes: 1
+  content: |
+   Ensure you install the following tools for building and deploying your apps.
+
+   | Install | For using... |
+   | --- | --- |
+   | [Visual Studio Code](https://code.visualstudio.com/download) or  [Visual Studio](https://visualstudio.microsoft.com/downloads/) | JavaScript, TypeScript, or CSharp build environments. Use the latest version. |
+   | [Teams Toolkit](https://marketplace.visualstudio.com/items?itemName=TeamsDevApp.ms-teams-vscode-extension) | Microsoft Visual Studio Code extension that creates a project scaffolding for your app. Use pre-release version of Teams Toolkit.|
+   | [Node.js](https://nodejs.org/en/download/) | Back-end JavaScript runtime environment. For more information, see [Node.js version compatibility table for project type](~/toolkit/build-environments.md#nodejs-version-compatibility-table-for-project-type).|
+   | [Microsoft Teams](https://www.microsoft.com/microsoft-teams/download-app) | Microsoft Teams to collaborate with everyone you work with through apps for chat, meetings, and call all in one place.|
+   | [OpenAI](https://openai.com/api/) or  [Azure OpenAI](https://oai.azure.com/portal)| First create your OpenAI API key to use OpenAI's GPT. If you want to host your app or access resources in Azure, you must create an Azure OpenAI service.|
+   | [Microsoft&nbsp;Edge](https://www.microsoft.com/edge) (recommended) or [Google Chrome](https://www.google.com/chrome/) | A browser with developer tools. |
+
+   ## Prepare development environment
+
+   After you've installed the required tools, set up the development environment.
+
+   ### Install the pre-release version of Teams Toolkit
```

Comment:
Install the stable version of Teams Toolkit instead?

Reply:
Removed this section and added Toolkit installation link in the table.